lumicks/pylake/tests/test_image.py

```diff
@@ -178,9 +186,11 @@ def test_percentile_ranges(minimum, maximum, ref_minimum, ref_maximum):
         ]
     )
     minimum, maximum = bounds.T
+    difference = maximum - minimum
+    difference[difference == 0] = 1  # Prevent div by zero
```

Comment:
Nit: I find this line confusing, even with the code comment.

Reply:
Changed it to:   `# don't scale components with a denominator of zero`